updated button ui/cards interface

acd37 opened the pull request:
- updated buttons [color, shadow etc. in line with MDL], inc. ST blue alignment
- updated dropdown menu border-radius, background-color, and height attributes
- MDL box-shadow attribute added to Logs and Variables panel
- responsive design improved on piston module

![screen shot 2017-12-14 at 9 36 15 pm](https://user-images.githubusercontent.com/25551773/34026964-e4f9d718-e116-11e7-8ce6-05b5f645c750.png)
![screen shot 2017-12-14 at 9 36 28 pm](https://user-images.githubusercontent.com/25551773/34026965-e512ee7e-e116-11e7-9c25-a5cbe9534a23.png)
![screen shot 2017-12-14 at 9 36 36 pm](https://user-images.githubusercontent.com/25551773/34026976-f04d0b12-e116-11e7-92a8-ebfc5f69e27f.png)

idpaterson commented:
This looks nice other than the following addition which will need to be modified:
```diff
+.col-md-4 {
+    border-style: 1px solid #eee;
+    box-shadow: 0 2px 5px 0 rgba(0, 0, 0, 0.16), 0 2px 10px 0 rgba(0, 0, 0, 0.12);
+    margin-top: 20px;
+    margin-bottom: 10px;
+    margin-left: 15px;
+    margin-right: 20px;
+    width: 30%;
+    min-width: 275px;
+    height: 100%;
+    min-height: 325px;
+}
```

`.col-md-4` is [a core Bootstrap class](http://getbootstrap.com/docs/3.3/css/#grid) that could be used anywhere in the app and normally applies only at certain screen widths. Changing its margins, size, and adding shadows will cause unexpected headaches down the road. If there is a specific part of the UI that you are targeting but no better class to target it with please add a class or additional markup as needed. I believe that in this case you would wrap the content of that column in a div with a meaningful class name that gets styled with the shadow. The outer `.col-md-4` would manage the width and margins.

Also, please [amend and force push to your branch](https://www.burntfen.com/2015-10-30/how-to-amend-a-commit-on-a-github-pull-request#option-2-manually-add-changes-amend-and-force-push) to fix the missing commit message so that we have a meaningful commit history.

acd37 commented:
Sounds good! I can do these when I land in the UK :)

On Fri, Dec 15, 2017 at 8:32 AM Ian Paterson <notifications@github.com>
wrote:

> This looks nice other than the following addition which will need to be
> modified:
>
> +.col-md-4 {+    border-style: 1px solid #eee;+    box-shadow: 0 2px 5px 0 rgba(0, 0, 0, 0.16), 0 2px 10px 0 rgba(0, 0, 0, 0.12);+    margin-top: 20px;+    margin-bottom: 10px;+    margin-left: 15px;+    margin-right: 20px;+    width: 30%;+    min-width: 275px;+    height: 100%;+    min-height: 325px;+}
>
> .col-md-4 is a core Bootstrap class
> <http://getbootstrap.com/docs/3.3/css/#grid> that could be used anywhere
> in the app and normally applies only at certain screen widths. Changing its
> margins, size, and adding shadows will cause unexpected headaches down the
> road. If there is a specific part of the UI that you are targeting but no
> better class to target it with please add a class or additional markup as
> needed. I believe that in this case you would wrap the content of that
> column in a div with a meaningful class name that gets styled with the
> shadow. The outer .col-md-4 would manage the width and margins.
>
> Also, please amend and force push to your branch
> <https://www.burntfen.com/2015-10-30/how-to-amend-a-commit-on-a-github-pull-request#option-2-manually-add-changes-amend-and-force-push>
> to fix the missing commit message so that we have a meaningful commit
> history.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ady624/webCoRE/pull/40#issuecomment-352034592>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AYXjnQ2znYcfolTPZw7P8dnRp-R32nF8ks5tApESgaJpZM4RDAUg>
> .
>
-- 

*Alec Down*

acd37 commented:
- moved css from app.css to piston module css file as it only pertains to the one html page
- created columnCards as ID rather than class for easy insertion to <div>
- corrected error where all information was wrapped in one card

idpaterson commented:
Merged into #57
